modules/sdk/gradle-plugins-target-platform/README.markdown

```diff
@@ -67,7 +69,7 @@ The plugin adds a series of configurations to your project:
 
 Name | Description
 ---- | -----------
-`targetPlatformBOMs` | Configures all the BOMs to import as managed dependencies.
+`targetPlatformBoms` | Configures all the BOMs to import as managed dependencies.
 `targetPlatformBundles` | Configures all the bundles in addition to the distro to resolve against.
```

Comment:
Hey @codyhoag  the `targetPlatformBundles` `targetPlatformRequirements` are now gone.  So we need to remove them from here.